docx/text/paragraph.py

```diff
@@ -421,7 +421,12 @@ def norm_left_indent(self):
             tab_cnt += 1
         if tab_cnt:
             if self.paragraph_format.tab_stops:
-                last_tab_stop = self.paragraph_format.tab_stops[tab_cnt-1]
+                try:
+                    last_tab_stop = self.paragraph_format.tab_stops[tab_cnt-1]
+                except IndexError:
+                    last_tab_stop_diff = tab_cnt - len(self.paragraph_format.tab_stops)
+                    last_tab_stop = self.paragraph_format.tab_stops[last_tab_stop_diff]
```

Comment:
@aleksandarbos  Did you mean here:
```python
                    last_tab_stop = self.paragraph_format.tab_stops[
                        len(self.paragraph_format.tab_stops) - 1]
```
?

Reply:
No, firstly i need to check if there are more tabs chars than tab stops (that's something that try/catch should handle). If there are more tab characters than tab stops, it will raise IndexError, where i need to determine the difference. 

also for this:
>  last_tab_stop = self.paragraph_format.tab_stops[
                        len(self.paragraph_format.tab_stops) - 1]

there's good way to get the latest tabstop: `self.paragraph_format.tab_stops[-1]` but in my case, i need to take care about tab chars as well.

Reply:
Ah, ok. Didn't know that `tab_stops` is an object with a proper list-like semantics. I still had issues with this piece of code. Will try to isolate a specific case. Maybe it was triggered by negative indents.

Reply:
Seems that it was triggered by negative indents problem.

Reply:
Woops. Sorry, I spoke too soon. Seems that there still are some problems. Investigating.

Reply:
I get `IndexError` on above line. The case is this empty paragraph with negative indent tab stop and two tab chars in the `2017 Ord 10 Marijuana in Multi Unit.docx` SMC ordinance.

![image](https://user-images.githubusercontent.com/1673425/53234466-df888480-368f-11e9-91b0-1d952c55388a.png)

`last_tab_stop_diff` gets the value of 1 while there is only 1 tabstop in the `tab_stops` list.

Reply:
haven't been expecting those. I've ignored such tab stops.